keyboards/terrazzo/config.h

```diff
@@ -0,0 +1,92 @@
+#ifndef CONFIG_H
+#define CONFIG_H
```

Comment:
```suggestion
#pragma once
```

Prefer `#pragma once` here over the old-style include guards.